filebeat/module/kafka/log/_meta/fields.yml

```diff
@@ -30,10 +31,12 @@
           description: >
             Java class the trace is coming from.
         - name: message
+          # Should this be mapped to `message` instead`? How is it different from the above message?
           type: text
           description: >
               Message part of the trace.
         - name: full
+          # Sounds like potentially worth as text, not sure where this fits
```

Comment:
This is almost exactly a dupe of `.trace.message` ([see here](https://github.com/elastic/beats/blob/31fec4f/filebeat/module/kafka/log/ingest/pipeline.json#L34-L37)). This should actually be removed from the events, IMO.

Reply:
agree, removing.

Reply:
https://github.com/elastic/beats/pull/10398